docs/qppe-server.yaml

```diff
@@ -999,6 +1025,28 @@ components:
           required: [ scoring_state, scoring_code, score ]
         - $ref: "#/components/schemas/Attempt"
 
+    PackageDependencies:
+      type: object
+      properties:
+        package_dependencies:
+          type: array
+          items:
+            type: object
+            properties:
+              namespace:
+                type: string
+                description: Namespace of a package.
+              short_name:
+                type: string
+                description: Short name of a package.
+              hash:
+                type: string
+                description: SHA256 hash of a package.
+            required: [ namespace, short_name, hash ]
+          description: Listing of all QuestionPy packages that were involved for this response. This information will
+            be used by the LMS to resolve a QPy-URI and other references.
+      required: [ package_dependencies ]
```

Comment:
Wäre es nicht einfacher (gewesen), nur das Array in ein neues Component zu stecken und dann jeweils
```yaml
package_dependencies:
  $ref: "#/components/schemas/PackageDependencies"
```
hinzuzufügen, statt mit `allOf` zu arbeiten?

Reply:
Stimmt, gefällt mir auch besser. Hab es abgeändert.